build.sbt

```diff
@@ -38,6 +41,8 @@ excludeDependencies ++= Seq(
 
 assemblyOutputPath in assembly := file("/tmp/conseil.jar")
 
+parallelExecution in Test := false
```

Comment:
this is already in master, you can merge this change before closing PR so it would be in the same place